src/index.ts

```diff
@@ -31,6 +31,7 @@ export { pipe } from './internal/util/pipe';
 export { noop } from './internal/util/noop';
 export { identity } from './internal/util/identity';
 export { isObservable } from './internal/util/isObservable';
+export { isPromise } from './internal/util/isPromise';
```

Comment:
So sorry, we don't want RxJS to be on the hook for supporting a utility function like this. I don't think we should be exporting thing.

I think that the tests and the other changes are valid, however, and I'd like to merge them.

Reply:
Makes sense, thank you. I'll get that change for you quick!